### ISSUE TITLE
[question] questions regarding nerf-related methods

### ISSUE THREAD

**Issue by simin75simin** [alicevision/meshroom#1688]
1. is it possible to add nerf-related methods to meshroom and keep the running time low?
2. is nerf better than meshroom now / in a couple of years?

thanks for answering.

**Comment by natowi** [comment]
1. yes
2. this depends on the purpose. I monitor the progress in the field [here](https://github.com/natowi/3D-Reconstruction-with-Deep-Learning-Methods). At the moment the DL methods are optimized for specific scenes/models and generally do not produce highly detailed models/pointclouds/depthmaps. But the existing methods can be efficient in combination with traditional photogrammetry methods. So far, traditional photogrammetry is (to my knowledge) superior to the DL methods regarding accuracy and detail. This difference will probably reduce over time.

The DL methods available at the moment that are actually good and available are most of the time licensed quite restrictive and/or the code is difficult to setup. I find it curios that so far, there is no ready to use executable to drop a few images in and get a model out. Yes, there are Colab notebooks, but if you want to run the code on your own computer you need to spend a considerable time to configure the environment. I tried to build some projects using pyinstaller to wrap all the dependencies up, but sadly I don´t have the time to configure my projects. Being able to test out some of the existing models and use them within Meshroom would be great and unique. If that is something you are considering working on, I could give you some pointers to promising projects that should not be to difficult (relatively speaking) to connect to Meshroom.

**Comment by simin75simin** [comment]
> If that is something you are considering working on, I could give you some pointers to promising projects that should not be to difficult (relatively speaking) to connect to Meshroom.

yeah i'd like to know could you show me some that will not slow down the runtime too much? thanks!

**Comment by natowi** [comment]
https://peterfalkingham.com/2022/04/26/neural-radiance-fields-nerf-and-instant-ngp-future-of-photogrammetry/

https://github.com/ventusff/neurecon

There are some projects that are OpenMVG (and therefore nearly Meshroom) compatible like https://github.com/AIBluefisher/EGSfM

There are a few more recent one, but I need to look them up.

**Comment by natowi** [comment]
https://github.com/joreeves/mr2nerf
